depends/packages/native_cxxbridge.mk

```diff
@@ -32,7 +32,7 @@ define $(package)_preprocess_cmds
 endef
 
 define $(package)_build_cmds
-  cargo build --locked --offline --release --package=cxxbridge-cmd --bin=cxxbridge
+  cargo build --offline --release --package=cxxbridge-cmd --bin=cxxbridge
```

Comment:
Why is `--locked` removed here?

Reply:
It's removed because upstream `cxx` is not consistent at updating their checked-in `Cargo.lock` file when they cut a new release. In the case of this latest release, the lockfile was updated just before the version bump, so it was correct except for the version of `cxx` itself. Hence by dropping `--locked` here while retaining `--offline`, we allow `cargo` to adjust the `Cargo.lock` file to work correctly with the published `cxx` version, while still preventing any lockfile updates due to network access (which is what we need to ensure for Gitian to work).

Reply:
Filed https://github.com/dtolnay/cxx/issues/1157